spec/request_handler/include_option_handler_spec.rb

```diff
@@ -2,26 +2,34 @@
 require 'spec_helper'
 require 'request_handler/include_option_handler'
 describe RequestHandler::IncludeOptionHandler do
+  let(:handler) do
+    options_type = Dry::Types['strict.string'].enum('user', 'email', 'user.posts')
```

Comment:
I think we should already convert to double underscores before comparing it to the allowed options so that the allowed options and defaults are consistent, otherwise the allowed options use the dot and the defaults use the double underscore version.